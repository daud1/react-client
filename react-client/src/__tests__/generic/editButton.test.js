import React from 'react';
import { shallow, render, mount } from 'enzyme';
import EditBtnMdl from '../../components/generic/editBtnMdl.component';
import shallowToJson from 'enzyme-to-json';

describe ('EditButtonModal Component', () => {
    it('renders without crashing', () => {
        const wrapper = shallow(<EditBtnMdl />);
        expect(wrapper.exists(<div></div>)).toBe(true);
    });

    it('matches snapshot', () => {
        const wrapper = shallow(<EditBtnMdl />);
        expect(shallowToJson(wrapper)).toMatchSnapshot();
    });
});