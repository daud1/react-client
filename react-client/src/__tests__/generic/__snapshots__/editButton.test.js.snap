// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`EditButtonModal Component matches snapshot 1`] = `
<div>
  <form
    onSubmit={[Function]}
  >
    <input
      name="name"
      placeholder="Enter new name"
      type="text"
    />
    <button
      type="submit"
    >
      Edit
    </button>
  </form>
</div>
`;
